README.md

```diff
@@ -115,7 +149,35 @@ console.log(issuer)
 
 ```
 
-A consuming app can use it directly using [did-jwt](https://github.com/uport-project/did-jwt) for verifying JWTs.
+Example of a verified JWT:
```

Comment:
This is a bit confusing as it is the output of verifyJWT. 

Show the JWT itself first and then the payload, explaining that it contains this